app/about/index.html

```diff
@@ -1,8 +1,5 @@
 ---
 layout: default
-title: DEVit Conference - Thessaloniki 20 May 2016
```

Comment:
@kbariotis you can overwrite the default page title if you add `title` in any page.
Take a look at https://github.com/skgtech/devit/blob/master/app/_layouts/default.html#L4